af/fastpath/fastpath/core.py

```diff
@@ -89,24 +89,18 @@ def setup():
     ap.add_argument("--noapi", action="store_true", help="Do not start API feeder")
     ap.add_argument("--stdout", action="store_true", help="Log to stdout")
     ap.add_argument("--db-uri", help="Database DSN or URI.")
-    ap.add_argument(
-        "--update",
-        action="store_true",
-        help="Update summaries and files instead of logging an error",
-    )
-    ap.add_argument(
-        "--stop-after", type=int, help="Stop after feeding N measurements", default=None
-    )
-    ap.add_argument(
-        "--no-write-to-db",
-        action="store_true",
-        help="Do not insert measurement in database",
-    )
-    ap.add_argument(
-        "--keep-s3-cache",
-        action="store_true",
-        help="Keep files downloaded from S3 in the local cache",
-    )
+    h = "Update summaries and files instead of logging an error"
+    ap.add_argument("--update", action="store_true", help=h)
+    h = "Stop after feeding N measurements"
+    ap.add_argument("--stop-after", type=int, help=h, default=None)
+    h = "Do not insert measurement in database"
+    ap.add_argument("--no-write-to-db", action="store_true", help=h)
+    h = "Keep files downloaded from S3 in the local cache"
+    ap.add_argument("--keep-s3-cache", action="store_true", help=h)
+    ap.add_argument("--ccs", help="Filter comma-separated CCs when feeding from S3")
+    h = "Filter comma-separated test names when feeding from S3 (without underscores)"
+    ap.add_argument("--testnames", help=h)
```

Comment:
Perhaps it's useful to add a docstring to the testnames argument mentioning that the test_name needs to be without the `_` (this is the case right?).

Reply:
It's written in the help message.

Reply:
Added to the function's docstring.